src/Tasks/Microsoft.NET.Build.Tasks/RuntimeGraphCache.cs

```diff
@@ -18,6 +18,12 @@ public RuntimeGraphCache(TaskBase task)
             _log = task.Log;
         }
 
+        public RuntimeGraphCache(IBuildEngine4 buildEngine4, Logger log)
```

Comment:
I don't think you need this anymore with the task refactoring.

Reply:
Reverted

Reply:
Actually, I still need this (PrepareForReadyToRunCompilation.cs line 89). We were chatting that we could leave the call to NuGetUtils.GetBestMatchingRid to be on the safe side

Reply:
Can't it call the existing TaskBase overload now?

Reply:
Ah, forgot that it's no longer a ToolTask. Fixed